src/coprocessor/dag/executor/aggregate.rs

```diff
@@ -298,6 +303,41 @@ impl AggrFunc for Extremum {
     }
 }
 
+struct Variance {
+    sum: f64,
+    variance: f64,
+    cnt: u64,
+}
+
+impl AggrFunc for Variance {
+    fn update(&mut self, ctx: &mut EvalContext, args: &mut Vec<Datum>) -> Result<()> {
+        if args.len() != 1 {
```

Comment:
these check is redundant and unnecessary, I think we can remove them..

Reply:
Why? Have we already checked then somewhere else?

Reply:
I think this check is necessary since we have not checked them before. Maybe we can check them in the initialization of the Aggregate functions one day, but not in this PR.